libnmstate/nm/connection.py

```diff
@@ -145,6 +147,11 @@ def safe_activate_async(self):
 
         cancellable = self._mainloop.new_cancellable()
 
+        if self.nmdevice and self.nmdevice.get_nm_plugin_is_missing():
+            raise NmstateMissingPluginError(
+                f'Missing NetworkManager plugin for device {self.devname}'
```

Comment:
Actually the error message should name the plugin, for example `f"The 'NetworkManager-ovs' plugin is missing but required to manage the device '{self.devname}'"

Reply:
I let you a comment below.